### DISCUSSION TITLE
An unexpected error occurred.

### DISCUSSION THREAD

**Discussion by TheWizardUk** [requarks/wiki#4102] [Help / Questions]
Hi

after following this guild https://docs.requarks.io/en/install/ubuntu

on each page load I am seeing "An unexpected error occurred."

![image](https://user-images.githubusercontent.com/49233984/122437766-edb1eb80-cf91-11eb-8d37-9d736947ed18.png)

also in the setting all the left hand menu is .title  example 
![image](https://user-images.githubusercontent.com/49233984/122438097-43869380-cf92-11eb-84a0-41cdf5ddadea.png)

Can someone help please?

**Comment by miukumac** [comment]
We ran into the same issue on our WikiJS Dockers - it seems the localization files are failing to load and things are breaking hard.

Triaged it by sideloading them and setting offline mode on for now.

**Comment by TheWizardUk** [comment]
Thanks for the reply miukumac 

can you details the steps to fix the issue please? i understand you can set it to off line mode in the config but what was side loaded to resolve the issue?

**Comment by miukumac** [comment]
Hi, I take it back - we are still getting errors on trying to save any page (  Cannot read property 'updated' of undefined ) despite embedding the locale files in the container.

So no, including the localization files in offline mode did not fix it completely :-(

**Comment by TheWizardUk** [comment]
How did you including the localization files in offline mode ?

**Comment by miukumac** [comment]
Here's a guide to doing so - if you are using Docker although the same works for plain install.
https://stackoverflow.com/questions/62383153/wiki-js-cant-go-through-corporate-proxy

Check the last reply.

**Comment by loyalstr** [comment]
sideload also being mentioned in the official guideline
[https://docs.requarks.io/install/sideload](https://docs.requarks.io/install/sideload)
